src/components/MapDetail/components/Timeline/Timeline.js

```diff
@@ -2,44 +2,139 @@ import React from 'react'
 import { Box, Text } from 'grommet'
 import styled from 'styled-components'
 
-const StyledSVG = styled.svg`
-  display: block;
+const Relative = styled(Box)`
+  position: relative;
 `
 
-export default function Timeline() {
+const Circle = styled(Box)`
+  border-radius: 50%;
+`
+
+const Year = styled(Text)`
+  position: absolute;
+  top: 16px;
+  left: 50%;
+  transform: translateX(-50%);
+`
+
+export const Slider = styled.input`
+  appearance: none;
+  position: absolute;
+  left: 2px;
+  top: 0;
+  margin: 0;
+  width: calc(100% - 4px);
+  height: 100%;
+  background: transparent;
+
+  &::-webkit-slider-thumb {
```

Comment:
Keep in mind browser specific styles won't work across the board. Here's how it looks in Firefox:

<img width="532" alt="Screen Shot 2021-03-30 at 12 06 23 PM" src="https://user-images.githubusercontent.com/5016731/113033729-f21af500-9156-11eb-947f-e4bad1fd76f0.png">

and Safari looks ok though:

<img width="518" alt="Screen Shot 2021-03-30 at 12 57 17 PM" src="https://user-images.githubusercontent.com/5016731/113034314-92711980-9157-11eb-9566-deddb049a98a.png">

Firefox likely can be fixed by duplicating the styles and applying it to their own specific browser prefix pseudoelement: https://developer.mozilla.org/en-US/docs/Web/CSS/::-moz-range-thumb